site/content/kapp-controller/docs/develop/app-spec.md

```diff
@@ -220,6 +220,16 @@ spec:
                 fieldPath: metadata.namespace
               - name: specificAnnotation
                 fieldPath: metadata.annotations['specificAnnotation']
+              # query for the version of the cluster
+              - name: kubeVersion
+                kubernetesVersion: {}
+              # query for all group versions (a list of the API groups and versions in the form "group/version") supported on this cluster
+              - name: apiVersions
```

Comment:
here it would be slightly more clear to say something like `apiGVs`  or `apiGroupsNRoses`  or anyways to include a suggestion in the name that it's both groups and versions.

Reply:
my thought process was that it's like `kubectl api-versions`; the output of that command is literally the same thing. I was also trying to convey that this is a user-authored string... so I tried to make it different than the Downward API field/key.

Reply:
ah, oh ... it's not our bad name.   So maybe that'll be least-surprising to people who are already well-versed in k8s lore.   wait - is that really our target user?

Reply:
spoke about this out-of-band. I'm sold on the idea of keeping it very very straight-forward. 👍🏻